packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -376,22 +376,22 @@ class RenderEditable extends RenderBox {
   }
 
   // This does not required the layout to be updated.
-  double get _preferredLineHeight => _textPainter.preferredLineHeight;
+  double get preferredLineHeight => _textPainter.preferredLineHeight;
```

Comment:
Please document whatever this is.
In particular, the documentation should include the following details:
* What the value represents.
* What purpose is served by exposing this value here.
* What the units of the value are.
* When the value might change.
* How to get notified when the value changes.